packages/ember-runtime/lib/mixins/array.js

```diff
@@ -60,6 +60,10 @@ export function objectAt(content, idx) {
   return typeof content.objectAt === 'function' ? content.objectAt(idx) : content[idx];
 }
 
+export function pushObject(arr, obj) {
+  return typeof arr.pushObject === 'function' ? arr.pushObject(obj) : arr.push(obj);
```

Comment:
Not sure if we should default to Array#push for this or if we should warn/error out here to let the developer know the prototype extension isn't on.

Reply:
to see this complete, `pushObject` must be implemented off of `arr` so that it can be used regardless of prototype extensions or not.